vc_vidyo/indico_vc_vidyo/api/client.py

```diff
@@ -77,11 +77,26 @@ def create_room_object(self, **kwargs):
 
     @raises_api_error
     def find_room(self, extension):
+        from indico_vc_vidyo.plugin import VidyoPlugin
         filter_ = self.client.factory.create('Filter')
         filter_.query = extension
+        filter_.limit = 40
         filter_.dir = 'DESC'
 
-        return self.soap.getRooms(filter_).room
+        counter = 0
+
+        while True:
+            filter_.start = counter*filter_.limit
```

Comment:
I know PEP8 leaves this unspecified but for the sake of consistency I'd still put spaces around the `*` - especially since in this case not using spaces does not improve readability (since it's only two operands)